pkgs/by-name/pa/papers/package.nix

```diff
@@ -85,12 +85,15 @@ stdenv.mkDerivation (finalAttrs: {
       gdk-pixbuf
       glib
       gtk4
+      gtksourceview5
```

Comment:
This is only needed because we forgot to add it to `propagatedBuildInputs` of `libspelling`.

Reply:
yeah I had noticed, I will add this as a fix.